.github/workflows/main.yml

```diff
@@ -36,13 +36,10 @@ jobs:
         sudo apt-get install libsndfile*
         sudo apt-get install xvfb
         python -m pip install --upgrade pip
-        pip install kenlm==0.1 --global-option="--max_order=12"
-        pip install psychopy==2023.2.1 --no-deps
         pip install attrdict3

```

Comment:
We could potentially add this in another install script and have the pip upgrade command as part of dev install